recipes/ocaml/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+export LC_ALL=C
```

Comment:
Can you please add a comment why this is needed?

Reply:
Again, suggested in their instructions - we could try it without and see what happens.

Reply:
I would go for without.